js/app.js

```diff
@@ -69,28 +67,32 @@ function postImages() {
  * @param {*} event Called upon submit
  */
 function handleVote(event) {
-  event.preventDefault(); //Prevent Page Refresh
   Product.iterationCounter++;
```

Comment:
The easiest way to deal with the votes-for-nothing problem is to wrap all of these in a condition... probably checking something about `event.target`